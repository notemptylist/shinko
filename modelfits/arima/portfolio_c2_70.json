{
    "numlags": 400,
    "results": [
        {
            "mean": -0.007236110718799133,
            "order": [
                3,
                0,
                1
            ],
            "rmse": 1.656306363197471,
            "tstamp": 1640209471.3888855
        },
        {
            "mean": -0.007236110718799133,
            "order": [
                2,
                0,
                3
            ],
            "rmse": 2.5782230856615675,
            "tstamp": 1640209471.3888855
        },
        {
            "mean": -0.007236110718799133,
            "order": [
                2,
                1,
                0
            ],
            "rmse": 2.7481329777054007,
            "tstamp": 1640209471.3888855
        },
        {
            "mean": -0.006439092603196155,
            "order": [
                5,
                0,
                1
            ],
            "rmse": 1.7064833493871696,
            "tstamp": 1640256538.680105
        },
        {
            "mean": -0.006439092603196155,
            "order": [
                4,
                0,
                3
            ],
            "rmse": 1.7827677059801663,
            "tstamp": 1640256538.680105
        },
        {
            "mean": -0.006439092603196155,
            "order": [
                4,
                3,
                1
            ],
            "rmse": 3.7527685844282423,
            "tstamp": 1640256538.680105
        }
    ],
    "stream": "portfolio_c2_70.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}