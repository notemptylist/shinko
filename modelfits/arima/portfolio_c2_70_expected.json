{
    "numlags": 400,
    "results": [
        {
            "mean": 0.05086305580245732,
            "order": [
                2,
                1,
                3
            ],
            "rmse": 0.5464863630797351,
            "tstamp": 1640139326.8252907
        },
        {
            "mean": 0.05086305580245732,
            "order": [
                5,
                1,
                1
            ],
            "rmse": 0.5480151209232718,
            "tstamp": 1640139326.8252907
        },
        {
            "mean": 0.05086305580245732,
            "order": [
                6,
                1,
                1
            ],
            "rmse": 0.548401608113635,
            "tstamp": 1640139326.8252907
        }
    ],
    "stream": "portfolio_c2_70_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            1
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            1
        ],
        [
            3,
            1,
            3
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}